src/main/resources/import.sql

```diff
@@ -1,4 +1,4 @@
-INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Introduction to Quarkus', true, 0, null)
-INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Hibernate with Panache', false, 1, null)
-INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Visit Quarkus web site', false, 2, 'https://quarkus.io')
-INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Star Quarkus project', false, 3, 'https://github.com/quarkusio/quarkus/')
+INSERT INTO todo(id, title, completed, ordering, url) VALUES (nextval('hibernate_sequence'), 'Introduction to Quarkus', true, 0, null);
```

Comment:
As indicated by @machi1990, the `;` seems now to be mandatory.

See https://github.com/quarkusio/quarkus/issues/3417#issuecomment-518092150

Reply:
This is also documented in the release-notes:

https://quarkus.io/blog/quarkus-0-20-0-released/#hibernate-orm-multi-line-support-in-import-scripts